src/components/Patient/LinkDepartmentsSheet.tsx

```
@@ -177,6 +177,29 @@ function DeleteOrganizationButton({
   );
 }
 
+function buildOrgPath(org: any): JSX.Element {
+  if (!org) return <></>;
+
+  const path: string[] = [];
+  let currentOrg = org;
+  while (currentOrg && currentOrg.name) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)